en/flight_controller/cuav_v5_plus.md

```diff
@@ -154,9 +154,11 @@ For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts of
 
 #### The HV\_PM power module output is unfused {#issue_pm_unfused}
 
+> **Warning** This is a serious safety issue.
+
 The *HV\_PM* power module supplied with the kit is not protected by a fuse:
```

Comment:
Strike this line.

Reply:
@davids5 I removed this line, but
- I am not sure why
- the heading still says it is unfused - is that a problem?